causalml/inference/tree/causal/_tree.py

```diff
@@ -42,7 +42,13 @@ def _support_missing_values(self, X) -> bool:
         return False
 
     def fit(
-        self, X, y, sample_weight=None, check_input=True, X_idx_sorted="deprecated"
+        self,
+        X,
+        y,
```

Comment:
@jeongyoonlee do we want to swap the order for `y` and `treatment` for consistency and also i see another lines are calling this function passing in the arguments in a different order? `tree.fit(X, treatment, y, sample_weight=curr_sample_weight, check_input=False)`

Reply:
Thanks, Paul. Good catch! I will update it with the fix.